README.md

```diff
@@ -1,77 +1,337 @@
 # React JSX
 
 ## Objectives
-
 1. Learn how to compile JSX to plain JavaScript
 2. Set up [browserify][browserify] and
    [babelify][babelify] for compiling your application.
 3. Render a React component using JSX
 
-## Overview
+## Introduction
+In this lesson, we'll be talking about using JSX in our React code, as well as how to transpile this code from JSX to
+JS that browsers understand. We'll do this by bundling our scripts using Browserify, and transforming the JSX using
+Babel. This lesson is fairly long, so strap in!
+
+## What's JSX?
+Simply put, JSX allows us to write XML in our JavaScript files. Wait, you say, we can already do this! We just save our
+XML in strings! Yep, but this is different. JSX is where JS and XML/HTML _meet_, creating a very special and extremely
+productive marriage.
+
+Up until now, we've been writing our components and elements using `React.createElement()`. JSX allows us to write
+these components in a way that is very similar to the end result we see in the browser (i.e. HTML), and is _much_ faster
+to write. It'll blow your mind — I promise.
+
+![Using JSX](https://media.giphy.com/media/l4HnT4tZzUozNFx4s/giphy.gif)
+
+## What does it look like?
+A React component written in JSX looks something like this:
+
+```js
+class Tweet extends React.Component {
+  render() {
+    return (
+      <div className="tweet">
+        <img src="http://twitter.com/some-avatar.png" className="tweet__avatar" />
+        <div className="tweet__body">
+            <p>We're writing this tweet in JSX. Holy moly!</p>  
+        </div>
+      </div>
+    );
+  }
+}
+```
+
+Isn't this great? Look how readable it is! It's _literally_ HTML, but in our JS! Looking at this code, there are some
+important things to note. First of all, JSX is _not_ a string — it's not inbetween quotes. Think of it as another type
+in JavaScript. Secondly, the parentheses containing the JSX are entirely optional, but recommended by convention.
+
+And las—
```

Comment:
Oh is las, like the word "last" being cut off? Maybe use a ellipsis instead?  "And las..."